packages/extension-starter/src/setup-app.ts

```diff
@@ -80,6 +82,8 @@ class Sample extends ApplicationExtension<Config> {
      */
     // TODO: do we need to pass in additional params like request.path? Get familiar with the SEO API.
     // TODO: how can developers call SCAPI? How can we pass in the api client (with the access token)?
+    // TODO: update .hbs template to add @CacheResult decorator
+    @CacheResult('_cachedRoutes')
```

Comment:
where in the codebase would I find the template used to create starter extensions using the script?